src/Core/MailTemplates/Handlebars/Layouts/TitleContactUs.html.hbs

```diff
@@ -0,0 +1,26 @@
+﻿{{#>FullHtmlLayout}}
+<div width="auto" cellpadding="0" cellspacing="0" style="padding:0; margin:-20px">
+    <div style="display: block; min-height: 204px; background-color: #175DDC; margin: 0;">
+        <div class="title-header-text" >
+            <div style="font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif; box-sizing: border-box; font-size: 24px; color: #ffffff; line-height: 32px; font-weight: 400; -webkit-font-smoothing: antialiased; -webkit-text-size-adjust: none;">
+                {{TitleFirst}}<b class="white-title">{{TitleSecondBold}}</b>{{TitleThird}}
+            </div>
+        </div>
+        <div class="title-header-image" style="margin-left: auto; margin-right: 0px; vertical-align: bottom;">
+            <img style="margin-left: auto; margin-right: 0px; display: block;" alt='' src='https://assets.bitwarden.com/email/v1/business.png' />
```

Comment:
Issue: We likely need to add alternate descriptions to the image so they can at least display text when they can't load

Reply:
I added a comment to the PR saying that i would add the alt text but after reading a little about it, the best practice for decorative images it to leave them as empty. 
Ref: 
https://accessibility.huit.harvard.edu/describe-content-images 
https://help.siteimprove.com/support/solutions/articles/80000863904-accessibility-image-alt-text-best-practices